Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -5,11 +5,21 @@
 using Xunit;
 using System.Collections.Generic;
 using System.Reflection;
+using Octokit.Internal;
 
 namespace Octokit.Tests.Conventions
 {
     public class ModelTests
     {
+        private static readonly HashSet<Type> ExcludedEnumTypes = new HashSet<Type>
```

Comment:
I excluded some enums from the test because they're not used in the actual API (I couldn't find documentation on them, at least. I think it might be worth crawling the request/response models and find enum types instead of looping through all in the assembly.